arangod/Cluster/ClusterTypes.h

```diff
@@ -41,6 +42,17 @@ typedef std::string ViewID;           // ID of a view
 typedef std::string ShardID;          // ID of a shard
 typedef uint32_t ServerShortID;       // Short ID of a server
 typedef std::string ServerShortName;  // Short name of a server
+namespace pmr {
+using ManagedString = std::basic_string<char, std::char_traits<char>,
+                                        ResourceUsageAllocator<char>>;
+typedef ManagedString ServerID;         // ID of a server
+typedef ManagedString DatabaseID;       // ID/name of a database
+typedef ManagedString CollectionID;     // ID of a collection
+typedef ManagedString ViewID;           // ID of a view
+typedef ManagedString ShardID;          // ID of a shard
+typedef uint32_t ServerShortID;         // Short ID of a server
```

Comment:
Do we need this extra typedef? It seems it does not need a special allocator for anything.
```suggestion
typedef uint32_t ServerShortID;         // Short ID of a server
```